[bazel build] Linking error due to not linking with pthread

When I compile gflags-example (as well as my own project), I get the following error:

```

robryk@iguatalpa:~/example$ bazel build ...
.
INFO: Found 1 target...
ERROR: /usr/local/google/home/robryk/example/foo/BUILD:1:1: Linking of rule '//foo:foo' failed: linux-sandbox failed: error executing command /usr/local/google/home/robryk/.cache/bazel/_bazel_robryk/0d0280d3a76251d432180c833ad92a5c/execroot/example/_bin/linux-sandbox ... (remaining 13 argument(s) skipped).
bazel-out/local-fastbuild/bin/external/gflags_git/libgflags.a(gflags.pic.o): In function `gflags_mutex_namespace::Mutex::Mutex()':
gflags.cc:(.text._ZN22gflags_mutex_namespace5MutexC2Ev[_ZN22gflags_mutex_namespace5MutexC5Ev]+0x39): undefined reference to `pthread_rwlock_init'
bazel-out/local-fastbuild/bin/external/gflags_git/libgflags.a(gflags.pic.o): In function `gflags_mutex_namespace::Mutex::Mutex(gflags_mutex_namespace::Mutex::LinkerInitialized)':
gflags.cc:(.text._ZN22gflags_mutex_namespace5MutexC2ENS0_17LinkerInitializedE[_ZN22gflags_mutex_namespace5MutexC5ENS0_17LinkerInitializedE]+0x3c): undefined reference to `pthread_rwlock_init'
bazel-out/local-fastbuild/bin/external/gflags_git/libgflags.a(gflags.pic.o): In function `gflags_mutex_namespace::Mutex::~Mutex()':
gflags.cc:(.text._ZN22gflags_mutex_namespace5MutexD2Ev[_ZN22gflags_mutex_namespace5MutexD5Ev]+0x2c): undefined reference to `pthread_rwlock_destroy'
bazel-out/local-fastbuild/bin/external/gflags_git/libgflags.a(gflags.pic.o): In function `gflags_mutex_namespace::Mutex::Lock()':
gflags.cc:(.text._ZN22gflags_mutex_namespace5Mutex4LockEv[_ZN22gflags_mutex_namespace5Mutex4LockEv]+0x20): undefined reference to `pthread_rwlock_wrlock'
bazel-out/local-fastbuild/bin/external/gflags_git/libgflags.a(gflags.pic.o): In function `gflags_mutex_namespace::Mutex::Unlock()':
gflags.cc:(.text._ZN22gflags_mutex_namespace5Mutex6UnlockEv[_ZN22gflags_mutex_namespace5Mutex6UnlockEv]+0x20): undefined reference to `pthread_rwlock_unlock'
collect2: error: ld returned 1 exit status
Target //foo:foo failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 3.682s, Critical Path: 0.18s
```

Adding `linkopts = [ "-lpthread" ]` to the binary target I'm compiling works the problem around for me.

--- a/BUILD
+++ b/BUILD
@@ -40,7 +40,7 @@ cc_library(
     includes = [
         "include",
     ],
-    linkopts = ["-pthread"],
+    linkopts = ["-lpthread"],
     visibility = ["//visibility:public"],
 )
 

--- a/BUILD
+++ b/BUILD
@@ -40,6 +40,7 @@ cc_library(
     includes = [
         "include",
     ],
+    linkopts = ["-pthread"],
     visibility = ["//visibility:public"],
 )
 

